scripts/datenlord-deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ -z "$1" ]
+then
+    kubectl apply -f datenlord-logging.yaml
+    kubectl apply -f datenlord-monitor.yaml
+else
+    helm repo add stable https://charts.helm.sh/stable
+    helm install prometheus stable/prometheus-operator --namespace monitoring
+    helm repo add elastic https://helm.elastic.co
+    helm install filebeat elastic/filebeat
+    helm install kibana elastic/kibana
+    helm install elasticsearch elastic/elasticsearch -f values.yaml
+fi
```

Comment:
Maybe change the name of this script to deploy-monitoring or others. It doesn't deploy DatenLord.